spec/controllers/api/v1/admin/user/evaluations_controller_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::Admin::User::EvaluationsController, type: :controller do
+  before do
+    allow(controller).to receive(:authenticate_user!).and_return(true)
+    allow(controller).to receive(:current_user).and_return(user)
+  end
+
+  let(:user) { FactoryGirl.create(:user, role: 'admin') }
+
+  describe "GET #index" do
+    it "returns success" do
+      FactoryGirl.create(:evaluation, user_id: user.id, goals_attributes: [attributes_for(:goal)])
```

Comment:
I think we already have `create` helper.

Reply:
also, it doesn't represent the logic in controller. In controller, you create evaluation in scope of user, not `current_user`